lib/Loop.lua

```diff
@@ -132,7 +132,7 @@ type System = (...any) -> () | { system: (...any) -> (), event: string?, priorit
 ]=]
 function Loop:scheduleSystems(systems: { System })
```

Comment:
Technically people can schedule a system more than once. A check could be doing `table.find(self._systems)` however we could also explore using a Map with the system mapped to the time of its entry allowing simple checks. Using os.clock there should be precise enough for each iteration of an entry to be unique.